Update OD- and HR-related articles with correct information (especially regarding osu!mania) (#6066)
ppy/osu-wiki: wiki/Beatmap_Editor/Song_Setup/en.md

> 

Reviewer: ```suggestion

*Main page: [Overall difficulty](/wiki/Beatmapping/Overall_difficulty)*

```

![Detailed OD chart](img/ODTable.png "Detailed OD chart")

> *Note: The above table only applies for osu!*

Reviewer: ```suggestion
*Note: The above table applies for the osu! game mode only.*
```

*Note: Overall difficulty is visible when viewing osu!catch beatmap information, but it does not affect gameplay.*

*Notice: On the beatmap page, Overall Difficulty is listed as Accuracy.*

Overall Difficulty, commonly abbreviated as OD, defines how difficult it is to achieve high accuracy. The values range from 0 to 10; higher OD values will require more accuracy and precision. Since accuracy is important for gaining HP, Overall Difficulty indirectly influences how hard it is to pass the beatmap. Higher OD values mean shorter time windows to clear the hit objects, both in general and terms of getting the highest score value from the hit object.

> In osu!, the timing system does not allow an object to be hit until the previous object is either (1) hit and receives a judgement or (2) missed after the hit window passes. If the hit windows is too long, the next object's hit window may overlap with the previous one. If an next object is hit in the overlapping time period while the previous object hasn't undergone (1) or (2), it will be "locked" and the hit will not count as a (1). This is informally called "notelock" in osu!, or "column locking" or "cbrushing" in osu!mania. For more details and examples, see the page [Notelock](/wiki/Gameplay/Notelock). For this reason, it is advised not to use low OD values in high BPM maps.

Author: I will simplify a bit and put the rest into Notelock then

Gamemode specifics:

- osu!: Spinners must be spun more to fill up the gauge.
- osu!taiko: The denden (spinner) require more hits to clear.

> - osu!mania: Each lane's timing window is independent and it is possible to notelock on any combination of columns.

Author: Say if there are 4 key columns then you can notelock on one column (constant misses because the judgement is calculated relative to previous note) while the other columns continue as usual (able to receive 300), that's what I mean for independent. For combination of columns I meant you can notelock on say col 2,4 while the rest is fine.

> - osu!mania: Each lane's timing window is independent and it is possible to notelock on any combination of columns.

Reviewer 2: So when OD is higher, you may gain more consist of misses?

> - osu!mania: Each lane's timing window is independent and it is possible to notelock on any combination of columns.

Reviewer 3: for the record, I removed this as irrelevant

- osu!catch is simply not affected by OD.

*Note: This does imply that raising the OD too high can make previously possible spinners near impossible to clear on time.*

> If you wish to calculate the hit windows, explicit formulas can be found [here](/wiki/Beatmapping/Overall_difficulty).

Reviewer: ```suggestion
If you wish to calculate the hit windows, explicit formulas can be found in the [main article on Overall Difficulty](/wiki/Beatmapping/Overall_difficulty).
```
